### DISCUSSION TITLE
Is it intended that the transaction be rolled back if writing to the output file specified with the -o option fails?

### DISCUSSION THREAD

**Discussion by shimizu-n** [microsoft/go-sqlcmd#540] [Q&A]
Is it intended that the transaction be rolled back if writing to the output file specified with the -o option fails?
This behavior is hard to understand.

**Comment by shueybubbles** [comment]
it might be a behavior where nobody ever wrote down what's supposed to happen. 
Are you able to compare go-sqlcmd behavior to that of ODBC sqlcmd? Are they different?
